apollo-portal/src/main/resources/static/audit_log_menu.html

```diff
@@ -0,0 +1,150 @@
+<!--
+  ~ Copyright 2023 Apollo Authors
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+-->
+<!doctype html>
+<html ng-app="audit_log_menu">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <link rel="icon" href="img/config.png">
+  <!-- styles -->
+  <link rel="stylesheet" type="text/css" href="vendor/bootstrap/css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="vendor/angular/angular-toastr-1.4.1.min.css">
+  <link rel="stylesheet" type="text/css" href="vendor/select2/select2.min.css">
+  <link rel="stylesheet" type="text/css" media='all' href="vendor/angular/loading-bar.min.css">
+  <link rel="stylesheet" type="text/css" media='all' href="vendor/font-awesome.min.css">
+
+  <link rel="stylesheet" type="text/css" href="styles/common-style.css">
+  <link rel="stylesheet" type="text/css" href="vendor/iconfont/iconfont.css">
+</head>
+<body>
+
+<apollonav></apollonav>
+<div class="container-fluid apollo-container" ng-controller="AuditLogMenuController">
+  <div id='audit-menu' class="panel">
+    <section class="panel-body">
+      <section class="row">
+        <div class="main-table">
+          <div class="search-bar">
+            <h1>{{'ApolloAuditLog.QueryTips' | translate}}</h1>
+            <form>
+              <div class="mb-3">
+                <label for="opName" class="form-label">{{'ApolloAuditLog.OpName' | translate}}</label>
+                <input type="text" class="form-control" id="opName" placeholder="{{'ApolloAuditLog.OpNameTips' | translate}}"
+                       ng-model="opName">
```

Comment:
Can we add an search + select input here?

for example, in http://localhost:8070/system-role-manage.html

<img width="689" alt="image" src="https://github.com/apolloconfig/apollo/assets/15523186/6fbf9157-f1cf-4363-9693-dd326bb78e96">

If it is hard, left it to next pr.

We can use select option instead search.

Reply:
Is this means we don't want to have the menu below and just show results in dropdown ?